src/sentry/lang/javascript/processor.py

```diff
@@ -153,7 +154,7 @@ def discover_sourcemap(result):
     sourcemap = result.headers.get("sourcemap", result.headers.get("x-sourcemap"))
 
     if not sourcemap:
-        parsed_body = result.body.split("\n")
+        parsed_body = result.body.split(b"\n")
```

Comment:
Shouldn't we be doing `result.body.decode(result.encoding).split("\n")` or something similar instead?